src/Command/AutomaticDeployCommand.php

```diff
@@ -130,15 +138,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     'pluginId' => $configuredPluginState->getId(),
                 ]);
                 $app->run($input, $output);
+            } else {
+                // Try to unactivate
+                $input = new ArrayInput([
+                    'command' => 'sd:plugins:deactivate',
+                    '--secure' => null,
```

Comment:
I think I have to check this first. I messed up something, I think this has to be:

`'secure' => true`

Reply:
OK, according to https://symfony.com/doc/current/console/calling_commands.html it has to be a mix of both:
`'--secure' => true`